ci/dev-pipeline.yml

```diff
@@ -88,6 +82,12 @@ resources:
       branch: main
       username: ((github_access_token))
       password: x-oauth-basic
+    ignore_paths:
+      - CHANGELOG.MD
+      - freeze.py
```

Comment:
are we ignoring these again? Think you 
mentioned it before

Reply:
this is to prevent automatic pipeline runs on commits with only these files updated, otherwise when you run preprod it will commit the updated changelog to main, which in turn triggers a fresh run on dev as commits would be detected, even worse if we made preprod automatic as well it would create an endless feedback loop of deployments